contributing.rst

```diff
@@ -4,16 +4,37 @@
 Contributing
 ============
 
-Flux Framework projects follow the Collective Code Construction Contract (`C4.1 <https://github.com/flux-framework/rfc/blob/master/spec_1.adoc>`_) which describes an optimal collaboration model for open source projects, based on the GitHub fork+pull model.
+The Flux Framework team welcomes all contributors for bug fixes, code improvements, new features, simplifications, documentation, and more. Please do not hesitate to `contact us <https://github.com/orgs/flux-framework/people>`_ with any questions or concerns.
 
-To contribute to a Flux Framework project:
+This guide details how to contribute to `Flux Framework projects <https://github.com/flux-framework>`_ in a standardized and efficient manner. Our projects follow the Collective Code Construction Contract (`C4.1 <https://github.com/flux-framework/rfc/blob/master/spec_1.adoc>`_) which describes an optimal collaboration model for open source projects, based on the GitHub fork+pull model.
 
-* Fork the project.
-* Clone your fork. ``git clone git@github.com:[username]/flux-framework/[project].git``
+.. _pull-requests:
+
+-------------
+Pull Requests
+-------------
+
+Use a pull request (PR) toward a repository's master branch to propose your contribution. If you are planning significant code changes, or have any questions, you can also open an issue before submitting a PR. To contribute to a Flux Framework project:
```

Comment:
Maybe:

> If you are planning significant code changes, are fixing a bug, or have any questions, please open...

Ideally, PR's reference an issue where a problem was reported, or where a new feature was proposed and discussed.  (This is a SHOULD not a MUST in our RFC 1).  So I think the language about opening an issue should be a little stronger but not dictatorial.